doc/source/splash.html

```diff
@@ -328,6 +328,7 @@ <h2 style="font-weight: 600">Scaling with Ray</h2>
             <a href="./serve/api/index.html" target="_blank">API references</a>
           </div>
           <div class="col-6" style="display: flex; justify-content: flex-end">
+            <!-- TODO: Update this to diffrent example. -->
             <a
               href="https://github.com/ray-project/ray/blob/master/doc/source/ray-air/examples/gptj_serving.ipynb"
```

Comment:
@akshay-anyscale can you choose a replacement for this?

Reply:
how about https://github.com/ray-project/ray/blob/master/doc/source/templates/03_serving_stable_diffusion/start.ipynb

Reply:
Nice, I replaced with https://github.com/ray-project/ray/blob/master/doc/source/serve/doc_code/stable_diffusion.py since it's what is shown in the Serve Examples page.